projects/scalar-app/src/main/index.ts

```diff
@@ -84,14 +84,19 @@ function createWindow(): void {
     return { action: 'deny' }
   })
 
-  // Disable CORS
+  // Allow cross-origin cookies
   mainWindow.webContents.session.webRequest.onBeforeSendHeaders((details, callback) => {
     const { requestHeaders } = details
 
-    upsertKeyValue(requestHeaders, 'Access-Control-Allow-Origin', ['*'])
```

Comment:
I don’t know how this ended up in the *REQUEST* headers, that’s a *RESPONSE* header.